### ISSUE TITLE
Dropdown Navigation 

### ISSUE THREAD

**Issue by arky** [plone/volto#2973]
**Is your feature request related to a problem? Please describe.**
The volto default navigation only shows the top level navigation items. If secondary navigation elements are available (Example: News items are good example https://localhost:3000/news/breaking-news) they are not shown as dropdown menu item or subitem in mobile navigation. 

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/808895/148895318-a21369d1-3ecf-4c73-b70a-f8df40673d86.png)

Implementation of such drop down is available here and in volto-dropdownmenu addon. 
https://water.europa.eu/freshwater
https://github.com/eea/volto-freshwater/blob/master/src/customizations/components/theme/Navigation/Navigation.jsx

**Describe alternatives you've considered**
You can override navigation as shown in the example solution or use available addons. But this feature is really basic (classic plone feature) that it needs to be shipped with volto.

**Additional context**
Full discussion of this feature in seen this community post. 
https://community.plone.org/t/volto-6-simple-dropdown-navigation/14685/4

**Comment by erral** [comment]
We have the following code that overrides Volto's Navigation component to achieve this:

```jsx
import React from 'react';
import NavItem from '@plone/volto/components/theme/Navigation/NavItem';
import { Dropdown } from 'semantic-ui-react';
import './dropdownmenu.css';
const NavItems = ({ items, lang }) => {
  return (
    <>

      {items.map((item) =>
            item && item.items && item.items.length > 0 ? (
              <Dropdown text={item.title} className="item" key={item.url}>
                <Dropdown.Menu key={item.url}>
                  {item.items.map((dropdownitem) => (
                    <NavItem item={dropdownitem} lang={lang} key={dropdownitem.url} />
                  ))}
                </Dropdown.Menu>
              </Dropdown>
            ) : (
              <NavItem item={item} lang={lang} key={item.url} />
            ),
          )}
    </>
  );
};

export default NavItems;

```

And the relevant CSS

```css 
i.icon,
i.icons {
  font-size: 1em !important;
}

```

**Comment by erral** [comment]
We can create an addon with this code and publish it without problems

**Comment by tiberiuichim** [comment]
No, I think it's better to have it shipped with Volto. @plone/volto-team Thoughts on this?

**Comment by giuliaghisini** [comment]
> No, I think it's better to have it shipped with Volto. @plone/volto-team Thoughts on this?

i approve

**Comment by erral** [comment]
We'll try to prepare a PR with this.

**Comment by nileshgulia1** [comment]
+1 for navigation dropdowns

**Comment by tisto** [comment]
I understand the need to ship with a more sophisticated menu solution in Volto. Though, I think we have to carefully consider what we ship with Volto by default. There are lots of options you can choose from when it comes to menus and it is not easy to choose a default option that fits all purposes.

Pure CSS solutions like the one we ship in Plone Classic have the big downside that they kick in even if you hover over the menu for a millisecond. They also don't work well for deeply nested structures that go beyond two levels (because you have to follow an exact mouse path to navigate through them).

Fat menus are a more up-to-date and more suitable solution for most larger websites and we have been using them for all our projects for years. The main advantage is that they work a lot better for jumps into the lower levels of the website (and therefore work better for larger websites). The Red Turtle folks created an add-on for this:

https://github.com/collective/volto-dropdownmenu

Though, there are lots of configuration options for a dropdown and also would require careful consideration.

My gut feeling is that it is better to not ship with a simple solution that can be extended than shipping with a bad solution like Plone Classic.

**Comment by tiberiuichim** [comment]
@tisto I think it's a basic use case that should be covered by default Volto. If indeed, we all customize Navigation one way or another, then it doesn't hurt to ship default with a minimal better feature to serve as the starting point for better integrator customization, or even just used as it is. As far as I can see from @erral's snippet, it is not a big addition in terms of code.

Hover-based menus are still a thing, I can point to https://www.ap.org/ or https://www.reuters.com/ for example. Even if this would be integrated today with Volto, it won't be a problem because on mobile we have the burger menu.

**Comment by erral** [comment]
We also have faced several issues with dropdown navigations, many of them accessibility related (show the sub-items on click or on hover, if it's on click what to do to go to the clicked item, if it is on hover what to do on touch-screens, ...).

We also think that a somehow simple dropdown menu (I don't know if 3 level like in classic, but at least a simple 2 level dropdown menu) should be present.

**Comment by arky** [comment]
I would strongly second @tiberiuichim [comment](https://github.com/plone/volto/issues/2973#issuecomment-1011925166). Providing multiple level navigation is common expectation of anyone customizing volto based site for personal or clients.
